Composer/packages/electron-server/src/appMenu.ts

```diff
@@ -257,10 +257,10 @@ export function initAppMenu(win?: Electron.BrowserWindow) {
         ['Cut', 'Copy', 'Delete'].forEach((label) => {
           const nativeModeId = label + '-native';
           const actionModeId = label;
-          menu.getMenuItemById(nativeModeId).visible = mode === 'native';
-          menu.getMenuItemById(actionModeId).visible = mode === 'action';
+          menu.getMenuItemById(nativeModeId)!.visible = mode === 'native';
```

Comment:
For known menu items the `getMenuItemById` function always returns the requested menu item. This change is needed due to Electron types update where they account for invalid values passed into the function.

Reply:
Seems fine to me since we should are not planning on changing these menu items anytime soon.